Doc/library/symtable.rst

```diff
@@ -214,6 +214,9 @@ Examining Symbol Tables
       Although ``A().f()`` raises :exc:`TypeError` at runtime, ``A.f`` is still
       considered as a method-like function.
 
+      .. deprecated-removed:: 3.14 3.16
```

Comment:
The deprecation should also be added to the 3.14 What's New in the "Deprecated" section.

Reply:
Oh yes, I forgot about it! thanks for that (doctest is still failing since the deprecation warning is shown, so I'll fix it either today or tomorrow)